src/assets/serverCalls/getVideos.js

```diff
@@ -0,0 +1,9 @@
+import axios from "axios"
+async function getVideos(){
+    try {
+
```

Comment:
you may remove this file if not needed ahead.

Reply:
I need this file for future